app/models/location.rb

```diff
@@ -373,6 +373,11 @@ def available_from
   end
 
   def status
+    return :incomplete if
+      postcode.blank? ||
+        units.blank? ||
+        type_of_unit.blank? ||
+        mobility_type.blank?
```

Comment:
Have you thought about how you might query for active locations? We have decided to calculate the status dynamically. Did you consider having a status column on locations that you update whenever a location is updated?

```suggestion
    return :incomplete if [postcode, units, type_of_unit, mobility_type].any?(&:blank?)
```

We can make this a little shorter with `any?` — what do you think?